src/cleanfigure.m

```diff
@@ -77,47 +81,25 @@ function cleanfigure(varargin)
   return;
 end
 % =========================================================================
-function indent = recursiveCleanup(meta, h, targetResolution, indent)
+function recursiveCleanup(meta, h, targetResolution, pruneText)
+    % Recursive function, that cleans up the individual childs of a figure
 
+    % Get the type of the current figure handle
     type = get(h, 'Type');
 
-    %display(sprintf([repmat(' ',1,indent), type, '->']))
-
-    % Don't try to be smart about quiver groups.
-    % NOTE:
-    % A better way to write `strcmp(get(h,...))` would be to use
-    %     isa(handle(h), 'specgraph.quivergroup').
-    % The handle() function isn't supported by Octave, though, so let's stick
-    % with strcmp().
-    if strcmp(get(h, 'Type'), 'specgraph.quivergroup')
```

Comment:
This might be related to HG1/Octave, not HG2 on which you are testing (where it indeed may be dead code).

I don't think we should remove it this hastily and certainly not mixed in with a lot of unrelated edits in the same commit.